components/footer.tsx

```diff
@@ -29,19 +29,19 @@ export default function Footer() {
                 <div className="flex flex-col items-center md:items-start space-y-3">
                   <Link
                     href="https://docs.starknet.id/"
-                    className="opacity-70 font-light transition-opacity"
+                    className="opacity-80 font-light transition-all duration-300 hover:opacity-55 hover:translate-y-[-2px]"
```

Comment:
Please remove the translate transition, we just want the opacity to vary

Reply:
changes implemented, removed the translate translation